lib/util/git/index.ts

```diff
@@ -223,18 +269,18 @@ export async function initRepo(args: StorageConfig): Promise<void> {
 
 async function resetToBranch(branchName: string): Promise<void> {
   logger.debug(`resetToBranch(${branchName})`);
-  await git.raw(['reset', '--hard']);
-  await git.checkout(branchName);
-  await git.raw(['reset', '--hard', 'origin/' + branchName]);
-  await git.raw(['clean', '-fd']);
+  await gitRetry(() => git.raw(['reset', '--hard']));
```

Comment:
reset and clean are also pure offline command and should not retried.

@rarkins WDYT?